isis/src/lro/apps/lronacpho/main.cpp

```diff
@@ -83,18 +88,40 @@ void IsisMain (){
     useBackplane = true;
   }
 
-  // Get parameters file
-  Pvl params(ui.GetFileName("PHOPAR"));
-  IString algoName =   PhotometricFunction::algorithmName(params);
-  algoName.UpCase();
+  // Get name of parameters file
+  QString calDir = "";
+  QString algoName = "";
+  QString algoFile = ui.GetAsString("PHOPAR");
 
-  // Use generic NAC algorithm
+  if(algoFile.toLower() == "default" || algoFile.length() == 0){
+    GetCalibrationDirectory("", calDir);
+    algoFile = calDir + "NAC_PHO_LROC_Empirical.????.pvl";
+  }
```

Comment:
You can embed this logic in the app XML, see how hical specifies this:

https://github.com/USGS-Astrogeology/ISIS3/blob/dev/isis/src/mro/apps/hical/hical.xml#L1485

Reply:
Thanks Jesse. Updated with suggested changes.